soccer/KickEvaluator.cpp

```diff
@@ -114,10 +114,10 @@ KickResults KickEvaluator::eval_pt_to_seg(Point origin, Segment target) {
     // No opponent robots on the field
     if (botMeans.size() == 0) {
         // Push it off to the side
-        botMeans.push_back(3);
+        botMeans.push_back(4);
```

Comment:
What is this line for?

Reply:
I didn't really need to change this, but the further away from 0, the better. We just can push it too far away otherwise the exp estimation will return a wrong value messing up the Gradient Ascent nodes.

To calculate the derivative of the kicking function, you need the smallest robot index or something. So if there are no opponent robots on the field, we need to just create a fake robot to place there. This just puts it off to the side further. So its 4 radians instead of 3 away from the center of the target. This also helps when we try and find the largest window. I find the largest difference between neighboring local maximum. So if the robot was in the middle of the target segment, it would require something a bit more complicated.